deps/v8/src/profiler/profile-generator.cc

```diff
@@ -805,14 +805,24 @@ void CpuProfileJSONSerializer::SerializeCallFrame(
     const v8::CpuProfileNode* node) {
   writer_->AddString("\"functionName\":");
   writer_->EscapeAndAddString(node->GetFunctionNameStr());
-  writer_->AddString(",\"lineNumber\":");
-  writer_->AddNumber(node->GetLineNumber() - 1);
-  writer_->AddString(",\"columnNumber\":");
-  writer_->AddNumber(node->GetColumnNumber() - 1);
+  if (node->GetLineNumber()) {
```

Comment:
Have you checked what GetLineNumber() returns in case of REPL? (the same applies to GetColumnNumber)

Reply:
This is an example of what it prints:
```
"callFrame":{"functionName":"","lineNumber":0,"columnNumber":12,"scriptId":162,"url":"REPL6"}
```